portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,44 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-20">
```

Comment:
Good idea! But apparently HTML actually only supports up to UTF-8 (HTML 5 can handle UTF-16) 😕 

I'd recommend running your HTML through an HTML validator like https://validator.w3.org/#validate_by_input There are a couple of HTML tags that are mismatched here that the validator can find for you.

It's not a big deal since the browser will still render the HTML correctly for you, but it's good practice to try and keep the HTML correct. I once went to an interview and the interviewer mentioned to me that my personal website's HTML was invalid lol

The browser will still render it fine, but it's good practice!

Reply:
Thanks for the review! I just pushed the requested changes to this same pull request as instructed. (Pull request updated).